Examples/Examples/FeatureFormExampleView.swift

```diff
@@ -17,103 +17,85 @@ import ArcGISToolkit
-                                editedTables.append(table)
-                                updateSubmitButtonVisibility()
-                            }
-                        }
-                }
-                .alert("Error", isPresented: alertIsPresented) {
-                } message: {
-                    if let error = alertError {
-                        Text(error)
-                    }
+                    featureFormView
                 }
-                .overlay {
-                    if isApplyingEdits {
-                        HStack(spacing: 5) {
-                            ProgressView()
-                                .progressViewStyle(.circular)
-                            Text("Applying edits")
-                        }
-                        .padding()
-                        .background(.thinMaterial)
-                        .clipShape(.rect(cornerRadius: 10))
-                    } else {
-                        EmptyView()
-                    }
+                .task(id: identifyScreenPoint) {
+                    guard !editsAreBeingApplied,
+                          let identifyScreenPoint else { return }
+                    await makeFeatureForm(point: identifyScreenPoint, map: mapView)
                 }
                 .toolbar {
-                    if submitButtonVisibility != .hidden {
-                        ToolbarItem(placement: .navigationBarTrailing) {
-                            Button("Submit") {
-                                Task {
-                                    await applyEdits()
-                                }
-                            }
-                        }
+                    ToolbarItem(placement: .navigationBarTrailing) {
+                        submitButton
                     }
                 }
         }
     }
 }
 
 extension FeatureFormExampleView {
+    /// An error encountered while submitting edits.
+    enum SubmissionError: LocalizedError {
+        case applyEdits(any Error)
+        case other(String)
+        
+        var errorDescription: String? {
+            switch self {
+            case .applyEdits(let error):
+                String(reflecting: error)
+            case .other(let message):
+                message
+            }
+        }
+    }
+    
     // MARK: Methods
     
-    /// Applies edits to the remote service.
+    /// Applies edits to the service feature table or geodatabase.
     private func applyEdits() async {
-        isApplyingEdits = true
-        defer {
-            isApplyingEdits = false
-            updateSubmitButtonVisibility()
-        }
+        editsAreBeingApplied = true
+        defer { editsAreBeingApplied = false }
         
         for table in editedTables {
             guard let database = table.serviceGeodatabase else {
-                alertError = "No geodatabase found."
+                submissionError = .other("No geodatabase found.")
```

Comment:
Instead of updating `submissionError` in multiple places in this method, it seems like it would make more sense to make `applyEdits()` throwing and have the caller update `submissionError` with the thrown error. That would also fit more with the spirit of guard, which is to exit the current scope without doing additional work.